packages/playground/src/index.js

```diff
@@ -1,620 +1,18 @@
-import React, { Component } from "react";
-import MonacoEditor from "react-monaco-editor";
-import { samples } from "./samples";
-import "react-app-polyfill/ie11";
-import Form, { withTheme } from "@rjsf/core";
-import DemoFrame from "./DemoFrame";
-
-// deepEquals and shouldRender and isArguments are copied from rjsf-core. TODO: unify these utility functions.
-
-function isArguments(object) {
-  return Object.prototype.toString.call(object) === "[object Arguments]";
-}
-
-function deepEquals(a, b, ca = [], cb = []) {
-  // Partially extracted from node-deeper and adapted to exclude comparison
-  // checks for functions.
-  // https://github.com/othiym23/node-deeper
-  if (a === b) {
-    return true;
-  } else if (typeof a === "function" || typeof b === "function") {
-    // Assume all functions are equivalent
-    // see https://github.com/mozilla-services/react-jsonschema-form/issues/255
-    return true;
-  } else if (typeof a !== "object" || typeof b !== "object") {
-    return false;
-  } else if (a === null || b === null) {
-    return false;
-  } else if (a instanceof Date && b instanceof Date) {
-    return a.getTime() === b.getTime();
-  } else if (a instanceof RegExp && b instanceof RegExp) {
-    return (
-      a.source === b.source &&
-      a.global === b.global &&
-      a.multiline === b.multiline &&
-      a.lastIndex === b.lastIndex &&
-      a.ignoreCase === b.ignoreCase
-    );
```

Comment:
are some of these functions common i.e. do we have deep equality in the lib etc?

happy but DRY?

Reply:
I've pointed out that there is a bug in this: https://github.com/rjsf-team/react-jsonschema-form/pull/1438#issuecomment-526318486

We still need to support this function until we can really refactor using an external lib. My recommendation is to convert everything to TS before we can tackle this.